server/src/test/java/org/elasticsearch/search/aggregations/bucket/filter/FilterAggregatorTests.java

```diff
@@ -104,7 +104,7 @@ public void testRandom() throws Exception {
                     response = searchAndReduce(indexSearcher, new MatchAllDocsQuery(), builder,
                         fieldType);
                 } else {
-                    response = search(indexSearcher, new MatchAllDocsQuery(), builder, fieldType);
+                    response = searchAndReduce(indexSearcher, new MatchAllDocsQuery(), builder, fieldType);
```

Comment:
I think we can remove the entire `if` block here, and the `doReduce` for loop.

Reply:
thanks, I pushed https://github.com/elastic/elasticsearch/pull/60683/commits/1dd98d16f924040dc870ab26c8feee62ad65e3ab